src/phases/query/QueryStatsVariants.tmpl

```diff
@@ -0,0 +1,157 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  This is a template of predefined variants for QueryStats workloads.
+
+GlobalDefaults:
+  # Local constants.
+  PhasesPerStage: &PhasesPerStage 6
```

Comment:
"stage" is different than "variant" - is this intentional? What is the difference if so? Could you add comments?